### PR TITLE
Feature/touch

### PR DESCRIPTION
**Pull request by ctacke**

### CODE REVIEW

File: Source/ProjectLab_Demo/MeadowApp.cs [hunk 1]
```diff
@@ -99,11 +99,25 @@ public override Task Initialize()
                 upButton.PressEnded += (s, e) => displayController.UpButtonState = false;
             }
```
Comment (adrianstevens) [inline Source/ProjectLab_Demo/MeadowApp.cs:101]: Need to guard this

File: Source/Meadow.ProjectLab/ProjectLabHardwareV3.cs [hunk 2]
```diff
@@ -360,4 +360,27 @@ public override ModbusRtuClient GetModbusRtuClient(int baudRate = 19200, int dat
     {
         return _connectors.GetModbusRtuClient(this, baudRate, dataBits, parity, stopBits);
     }
+
+    /// <inheritdoc/>
+    public override ITouchScreen? Touchscreen
```
Comment (adrianstevens) [inline Source/Meadow.ProjectLab/ProjectLabHardwareV3.cs:365]: We should move this into a v4 class but great to leave it for now

Comment (ctacke) [reply Source/Meadow.ProjectLab/ProjectLabHardwareV3.cs:365]: The hardware I have with a touchscreen is 3.f, so.....

Comment (jorgedevs) [reply Source/Meadow.ProjectLab/ProjectLabHardwareV3.cs:365]: @ctacke Bryan confirmed he's bumped the PL with touchscreen to v4, it'll show up in the upcoming board revisions.

**Review by jorgedevs — APPROVED** [review]

**Pull request merged**